src/commands/synthetics/README.md

```diff
@@ -155,6 +155,7 @@ For example:
   "tunnel": true
 }
 ```
+**Note**: A `global` filed was present in the global configuration file. This field is now deprecated and replaced by `defaultTestOverrides`.
```

Comment:
Suggestion:
```suggestion
**Note**: The `global` field from the global configuration file is now deprecated in favor of `defaultTestOverrides`.
```

Reply:
But let's wait what the documentation team thinks

Reply:
The suggestion looks good to me, but I'd remove the word "now"